src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpContentJsonExtensions.cs

```diff
@@ -89,7 +89,7 @@ public static partial class HttpContentJsonExtensions
         [RequiresDynamicCode(SerializationDynamicCodeMessage)]
         private static async Task<object?> ReadFromJsonAsyncCore(HttpContent content, Type type, JsonSerializerOptions? options, CancellationToken cancellationToken)
         {
-            using (Stream contentStream = await GetContentStreamAsync(content, cancellationToken).ConfigureAwait(false))
+            await using (Stream contentStream = await GetContentStreamAsync(content, cancellationToken).ConfigureAwait(false))
```

Comment:
This will not use .ConfigureAwait(false) on the implicit DisposeAsync call. The code would need to be changed to include that.

Reply:
https://github.com/dotnet/csharplang/discussions/2661 https://github.com/dotnet/csharplang/discussions/8778

Currently `await using` is basically unusable under the requirement of `ConfigureAwait`.